models/product-type/src/generator.ts

```diff
@@ -2,20 +2,21 @@ import * as AttributeDefinition from '@commercetools-test-data/attribute-definit
 import { ClientLogging } from '@commercetools-test-data/commons';
 import { fake, Generator } from '@commercetools-test-data/core';
 import { createRelatedDates } from '@commercetools-test-data/utils';
-import { TProductType } from './types';
+import { TProductTypeDefault } from './types';
 
 const [getOlderDate, getNewerDate] = createRelatedDates();
 
 // https://docs.commercetools.com/api/projects/productTypes#product-types
 
-const generator = Generator<TProductType>({
+const generator = Generator<TProductTypeDefault>({
   fields: {
     id: fake((f) => f.datatype.uuid()),
     version: fake((f) => f.datatype.number()),
     key: fake((f) => f.lorem.slug()),
     name: fake((f) => f.name.fullName()),
     description: fake((f) => f.random.words()),
     attributes: fake(() => [AttributeDefinition.random()]),
+    attributeDefinitions: fake(() => [AttributeDefinition.random()]),
```

Comment:
This is where I'm confused - attributeDefinitions should *not* be the same shape as attributes, correct?

Reply:
That's correct, it's in the `Result` shape:
![image](https://user-images.githubusercontent.com/8136478/220655976-22019adc-6289-491c-bebe-9ca8e3db1426.png)
![image](https://user-images.githubusercontent.com/8136478/220656050-0ed3dc02-e396-4213-8a9d-7d0d4fb63462.png)

Reply:
I think I understand. And we don't need limit/offset/total here, so 
```
    attributes: fake(() => [AttributeDefinition.random()]),
    attributeDefinitions: fake(() => {
      return {
          results: [AttributeDefinition.random()],
          __typename: 'AttributeDefinitionResult',
        }
    }),
```